elements/pfe-markdown/demo/pfe-markdown.html

```diff
@@ -0,0 +1,61 @@
+<link rel="stylesheet" href="/core/pfe-styles/red-hat-font.min.css">
+<link rel="stylesheet" href="/core/pfe-styles/pfe-base.css">
+<link rel="stylesheet" href="/core/pfe-styles/pfe-context.css">
+<link rel="stylesheet" href="/core/pfe-styles/pfe-layouts.css">
+<link rel="stylesheet" href="/core/pfe-core/pfe.min.css">
```

Comment:
Not sure these files need to be here, I think they can be removed.

```Suggestion
 
```

Reply:
did you check `http://localhost:8080/components/markdown/demo/`?